src/absil/il.fs

```diff
@@ -3169,10 +3169,13 @@ let mkILNestedExportedTypesLazy (l:Lazy<_>) =
 
 let mkILResources l =  ILResources l
 
-let addMethodImplToTable y tab =
+let addMethodImplToTable y (tab:Map<_,_>) =
     let key = (y.Overrides.MethodRef.Name, y.Overrides.MethodRef.ArgTypes.Length)
-    let prev = Map.tryFindMulti key tab
-    Map.add key (y::prev) tab
+    match tab.TryGetValue key with
```

Comment:
I'd rather not make this kind of change.  The declarative `tryFindMulti` has been expanded out into larger code, it's harder to read and think about.  The adjustment should be in tryFindMulti.